pythoncz/models/github.py

```diff
@@ -82,9 +82,9 @@ def _enhance_issue(session, issue):
     try:
         reactions = issue_details['reactions']
         issue['votes'] = (
-            reactions.get('total_count', 0)
-            - reactions.get('-1', 0)
-            - reactions.get('confused', 0)
+            reactions.get('total_count', 0) -
+            reactions.get('-1', 0) -
+            reactions.get('confused', 0)
```

Comment:
I find this quite silly...
https://www.python.org/dev/peps/pep-0008/#should-a-line-break-before-or-after-a-binary-operator